invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/user_dashboard/uploads.js

```diff
@@ -143,9 +143,9 @@ export const RDMRecordResultsListItem = ({ result, index }) => {
         <Item.Header as="h2">
           <a href={viewLink}>{title}</a>
         </Item.Header>
-        <Item.Meta className="creatibutors">
+        <Item className="creatibutors">
```

Comment:
Why was the `.Meta` removed? It seems to remove the top margin if changed to just `Item`?

Reply:
It adds the meta and ui classes, and some styles cascading to the children that aren't present for creatibutors lists elsewhere. To keep it, we could override or otherwise take them into account, and then add "Meta" wherever there is a creatibutor list.